command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -312,6 +312,38 @@ def _run_look(self, spotings_req):
 
             yield self.sub.nh.sleep(.5)
 
+class PoseSequenceCommander(object):
+    def __init__(self, sub):
+        self.sub = sub
+
+    @util.cancellableInlineCallbacks
+    def go_to_sequence_eulers(self, positions, orientations, speed=0.2):
+        '''Pass a list of positions and orientations (euler).
+        Each is realive to the sub's pose folloing the previous
+        pose command.
+        '''
+        for i in xrange(len(positions)):
+            yield self.sub.move.look_at_without_pitching(np.array(positions[i][0:3])).go(speed)
```

Comment:
Why do we look at the point first? I think the desired behavior is one move setting relative position and orientation from the current position. This lets the controller pick how to get to this pose.
```
pose = self.sub.pose
yield self.sub.move.relative(np.array(positions[0:3])).set_orientation(transformations.quaternion_multiply(
    self.sub.pose.orientation,
    transformations.quaternion_from_euler(orientations[i][0], orientations[i][1], orientations[i][2]))).go(speed)
```